x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/rest/RestInferenceAction.java

```diff
@@ -33,7 +33,12 @@ public List<Route> routes() {
     }
 
     @Override
-    protected ActionListener<InferenceAction.Response> listener(RestChannel channel) {
+    protected boolean shouldStream() {
```

Comment:
This maps to `_inference/<optional task type>/<inference endpoint id>` which is always non-streaming.